Azure Maps Search (#29350)
Azure/azure-sdk-for-net: sdk/maps/Azure.Maps.Common/samples/README.md

eng\scripts\CodeChecks.ps1 -ServiceDirectory maps---
page_type: sample
languages:
- csharp
products:
- azure
name: Azure.Maps.Common samples for .NET
description: Samples for the Azure.Maps.Common client library.
---

# Azure.Maps Samples

## Import the namespaces

To use common module, import `Azure.Maps` namespace:

```C# Snippet:ImportCommonNamespace
using Azure.Maps;
```

> ## Functionalities

Reviewer: ```suggestion
## Features
```

nit:  "Functionalities" gets flagged for grammar.    "Functionality" would work but there is no plural.

> Below is the functionalities we have in Azure.Maps.Common module:

Reviewer: ```suggestion
Below are the features of the Azure.Maps.Common module:
```
